Source/AliveLibAO/Animation.cpp

```diff
@@ -907,7 +911,8 @@ signed __int16 Animation::Init_402D20(int frameTableOffset, DynamicArray* /*anim
     }
 
     // NOTE: OG bug or odd compiler code gen? Why isn't it using the passed in list which appears to always be this anyway ??
-    if (!gObjList_animations_505564->Push_Back(this))
+    const auto result = gObjList_animations_505564->Push_Back(this);
+    if (!result)
     {
         return 0;
```

Comment:
really this should never happen, perhaps we should log an error here (maybe it will appear when stuff vanishes)